.github/workflows/manual.yml

```diff
@@ -0,0 +1,30 @@
+name: "Build NixOS manual"
+
+on:
+  pull_request_target:
+    paths:
+      - 'nixos/**'
```

Comment:
This wont build the nixpkgs manual.

Reply:
sorry. missed that part.

Reply:
I reverted and only added the restrict-eval part.